<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор тестов</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Опции</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Новая задача</string>
     </property>
     <addaction name="create_multi"/>
     <addaction name="create_single"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="separator"/>
    <addaction name="exit_action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="delete_action">
   <property name="text">
    <string>Удалить задачу</string>
   </property>
  </action>
  <action name="exit_action">
   <property name="text">
    <string>Выйти из редактора</string>
   </property>
  </action>
  <action name="create_multi">
   <property name="text">
    <string>Несколько ответов</string>
   </property>
  </action>
  <action name="create_single">
   <property name="text">
    <string>Один ответ</string>
   </property>
  </action>
  <action name="save_action">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
